rust/connlib/tunnel/src/device_channel/tun_windows.rs

```diff
@@ -258,7 +271,7 @@ fn start_recv_thread(
                     }
                 }
             }
-            tracing::debug!("recv_task exiting gracefully");
+            tracing::info!("recv_task exiting gracefully");
```

Comment:
This will be logged in all cases where the `loop` exits which also happens in the case of an error. Can we re-structure this so we only have 1 exit log in each branch?

You could for example `break` with a reason:

```rust
let reason = loop {
	break "packet channel closed";
};
```

Reply:
Do we need to have these logs on `info`? It feels more like `debug` to me as in, if everything works normally, we don't want to see these?

Reply:
Yeah if the deadlock is gone completely it should be `debug`, but for now let's keep it `info` so we can see if it's still there without having customers change to debug